### DISCUSSION TITLE
[Q&A] Android PWA 键盘模式下无法点击发送按钮解决方法

### DISCUSSION THREAD

**Discussion by leeyiding** [lobehub/lobehub#311] [General Question] [answered]
### 💻 系统环境 | Operating System

Windows

### 🌐 浏览器 | Browser

Other

### 🐛 问题描述 | Bug Description

在PWA中，开启键盘的情况下，没办法点发送按钮，只能先把键盘关了才行

https://github.com/lobehub/lobe-chat/assets/49098278/ca18b2b7-a416-4950-b2f7-86a009899cdf

**Comment by leeyiding** [comment]
浏览器中倒是没这个问题

**Comment by canisminor1990** [comment]
你重新安装下 PWA 试试呢，我当前测试环境是安卓+搜狗输入法，用手机键盘自带的回车，和输入栏上的发送都能一次触发

**Comment by canisminor1990** [comment]
- 第一次为手机键盘触发
- 第二次为输入栏按钮触发

https://github.com/lobehub/lobe-chat/assets/17870709/ea01b926-a93c-4975-a776-0d204c2fde92

**Comment by leeyiding** [comment]
我又重装了一次，下面区域内的按钮，包括输入框都不可以点击

跟输入法应该没关系，我测试Gboard也是这样的

https://github.com/lobehub/lobe-chat/assets/49098278/9af37c60-2433-42d5-be8d-602a2c670225

**Comment by canisminor1990** [comment]
安卓版本，和 chrome 版本多少

**Comment by leeyiding** [comment]
如果实在没办法复现修复的话，我继续用网页版好了

![S31012-16251438_com android chrome](https://github.com/lobehub/lobe-chat/assets/49098278/47f27ed8-7de0-4f5c-9878-26894f5ccc24)

**Comment by canisminor1990** [comment, marked as answer]
更新至 chrome 118 , 重新安装PWA

**Comment by leeyiding** [comment]
在更新到 118.0.5993.65 版本，并且清除数据重新安装PWA后成功解决了，非常感谢
